adoc/bp_chap_expanded_support.adoc

```diff
@@ -430,72 +428,99 @@ Some actions are not yet supported:
+* Repository Type: deb
 +
-----
-<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
-----
+For `main-updates`:
 
-.Procedure: Preparing an {Ubuntu} 18.04 Client for Registration
+* Repository Label: ubuntu-bionic-main-updates
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic-updates/main/binary-amd64/
+* Repository Type: deb
 
-. On the client, open the [filename]``/etc/apt/sources.list.d/suma_client_tools.list`` file, and add this line:
+. Create custom channels under the `pool` channel, mirroring the vendor channels.
+For more information on creating custom channels, see xref:modules/administration/pages/channel-management.adoc[].
+Ensure the custom channels you create have `AMD64 Debian` architecture.
++
+For example:
 +
 ----
-deb http://download.suse.de/ibs/Devel:/Galaxy:/Manager:/3.2:/Ubuntu18.04-SUSE-Manager-Tools/xUbuntu_18.04/ /
+ ubuntu-18.04-pool for amd64 (vendor channel)
+ |
+ +- Ubuntu-18.04-SUSE-Manager-Tools for amd64 (vendor channel)
+ |
+ +- ubuntu-18.04-amd64-main (custom channel)
+ |
+ +- ubuntu-18.04-amd64-main-updates (custom channel)
 ----
-+
-. Edit the [filename]``sudoers`` file:
-+
+. Associate the custom channels with the appropriate custom repositories.
+. Synchronize the new custom channels.
+You can check the progress of your synchronization from the command line with this command:
 ----
-sudo visudo
+tail -f /var/log/rhn/reposync.log /var/log/rhn/reposync/*
 ----
-+
-Grant [command]``sudo`` access to the user by adding this line to the [filename]``sudoers`` file. Replace [systemitem]``<user>`` with the name of the user that will be used to bootsrap the client in the {webui}:
+. To use bootstrap with {ubuntu}, you will need to create a bootstrap repository.
+You can  do this from the command line with [command]``mgr-create-bootstrap-repo``:
 +
 ----
-<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
+mgr-create-bootstrap-repo --with-custom-channels
 ----
 
 
+The root user on {ubuntu} is disabled by default.
+You can enable it by editing the [filename]``sudoers`` file.
 
-.Procedure: Synchronizing {ubuntu} Repositories
-
+.Procedure: Granting Root User Access
 
-. In the {susemgr} {webui}, navigate to menu:Software[Manage > Repositories] and click btn:[Create Repository].
-. In the [guimenu]``Create Repository`` dialog, use these values to create a new repository:
+. On the client, edit the [filename]``sudoers`` file:
 +
-* In the [guimenu]``Repository URL`` field, type the URL that contains the appropriate binary files, for example [path]``http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/``.
-* In the [guimenu]``Repository Type`` field, select [systemitem]``deb``.
+----
+sudo visudo
+----
++
+Grant [command]``sudo`` access to the user by adding this line to the [filename]``sudoers`` file. Replace [systemitem]``<user>`` with the name of the user that will be used to bootsrap the client in the {webui}:
```

Comment:
sorry for being late...

s/bootsrap/bootstrap

Reply:
Oops! Committed the fix directly to 3.2.